supplementary/alica_ros1/alica_tests/src/test/Expr/include/alica_tests/Behaviour/SuccessOnInitBeh3821787310391665935.h

```diff
@@ -0,0 +1,14 @@
+
+
+#include <alica_tests/DomainCondition.h>
+/*PROTECTED REGION ID(incBC3821787310391665935) ENABLED START*/
+// Add additional includes here
+/*PROTECTED REGION END*/
+
+namespace alica
+{
+/*PROTECTED REGION ID(meth3821787310391665935) ENABLED START*/
+// Add additional includes here
+/*PROTECTED REGION END*/
+
+} /* namespace alica */
```

Comment:
Maybe we can remove unused domain conditions from codegen for behaviours as well in a different PR

Reply:
yeah. Even better, we get rid of code gen entirely soon